project_script/main.py

```diff
@@ -1,108 +1,105 @@
-        time_photo_celiv_ramp_time = equalize_data_frame_rows(time_photo_celiv_ramp_time, even_columns_subtracted)
+    if dt.is_bigger_data_frame(
+            time_photo_celiv_ramp_time,
+            even_columns_subtracted
+    ):
+        time_photo_celiv_ramp_time = dt.equalize_data_frame_rows(
+            time_photo_celiv_ramp_time,
+            even_columns_subtracted
+        )
         return
-    even_columns_subtracted = equalize_data_frame_rows(even_columns_subtracted, time_photo_celiv_ramp_time)
+    even_columns_subtracted = dt.equalize_data_frame_rows(
+        even_columns_subtracted,
+        time_photo_celiv_ramp_time
+    )
 
 
 def integrate_data():
     global time_photo_celiv_ramp_time_transposed, even_columns_subtracted_transposed_in_arrays
     integrated_values = []
     for key, value in enumerate(even_columns_subtracted_transposed_in_arrays):
-        integrated_values.append(integrate.simps(even_columns_subtracted_transposed_in_arrays[key],
-                                                 time_photo_celiv_ramp_time_transposed[key], even='avg'))
+        integrated_values.append(
+            integrate.simps(
+                even_columns_subtracted_transposed_in_arrays[key],
+                time_photo_celiv_ramp_time_transposed[key],
+                even='avg'
+            )
+        )
     return integrated_values
 
 
-# device_thickness = ask.ask_device_thickness()
+# device_thickness = ask.device_thickness()
+# device_area = ask.device_area()
+charge_density_calculation_constant = cv.current_correction_factor * cv.charge_density_correction_factor \
+                                      / (cv.electron_charge * cv.device_area * cv.device_thickness)
 
-# device_area = ask.ask_device_area()
-
-electron_charge = 1.6*10**(-19)
-
-device_area = 1.4*10**(-5)
-
-device_thickness = 600*10**(-9)
-
-current_correction_factor = 1*10**(-9)
-
-charge_density_correction_factor = 1*10**(-6)
-
-charge_density_calculation_constant = current_correction_factor * charge_density_correction_factor \
-                                      / (electron_charge * device_area * device_thickness)
-
-# path_dark = ask.ask_file_path("dark-CELIV")
+# path_dark = ask.file_path("dark-CELIV")
 #
 # while not is_valid_file(path_dark):
 #     message_invalid_path()
-#     path_dark = ask.ask_file_path("dark-CELIV")
+#     path_dark = ask.file_path("dark-CELIV")
 #
-current_dark_celiv_ramp_time = separate_even_columns(pd.read_table("C:/Users/robee/Desktop/"
-                                                                   "2-dark-celiv-current-celiv-only-alterado.txt",
-                                                                   sep='\t', header=None))\
+current_dark_celiv_ramp_time = dt.separate_even_columns(
+    pd.read_table(
+        "C:/Users/robee/Desktop/2-dark-celiv-current-celiv-only-alterado.txt",
+        sep='\t',
+        header=None
+    )
+)\
     .abs()\
     .dropna()
```

Comment:
Maybe the correct indentation would be like this, right?

![image](https://user-images.githubusercontent.com/11571434/156576850-bed79a7e-2140-4e2f-87ad-c13443b5c50d.png)